src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -1951,9 +1950,19 @@ public void IdleOperations(object state)
                         if (idleDbInstance.Operations.HasActive)
                             continue;
 
+                        var dbIdEtagDictionary = new Dictionary<string, long>();
+                        using (idleDbInstance.DocumentsStorage.ContextPool.AllocateOperationContext(out DocumentsOperationContext documentsContext))
+                        using (documentsContext.OpenReadTransaction())
+                        {
+                            foreach (var kvp in DocumentsStorage.GetAllReplicatedEtags(documentsContext))
+                            {
+                                dbIdEtagDictionary[kvp.Key] = kvp.Value;
+                            }
+                        }
+                        IdleDatabases[idleDbInstance.Name.ToLowerInvariant()] = dbIdEtagDictionary.Count > 0 ? dbIdEtagDictionary : null;
```

Comment:
What is bad about putting empty dic here?

Reply:
done